editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -502,7 +502,7 @@ void Node3DEditorViewport::_select_clicked(bool p_allow_locked) {
 	}
 }
 
-ObjectID Node3DEditorViewport::_select_ray(const Point2 &p_pos) {
+ObjectID Node3DEditorViewport::_select_ray(const Point2 &p_pos) const {
```

Comment:
Adding `const` to methods is good but as with the comment above it could be separate.

Reply:
Yeah, it can't be called from the 'can_drop_data_fw' because of const requirement